app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import NotVaccinatedError, \
+    OutdatedVaccineError, NotWearingMaskError
```

Comment:
it is better to take all classes in parentheses
so you can specify each class from the next line and not use "\" to move to the next line
and it will look better